### PR TITLE
chore: create intake forms for bugs & enhancements

### PR DESCRIPTION
**Pull request by NWRichmond**
### WHY are these changes introduced?

Addresses [LS-1348](https://nacelle.atlassian.net/browse/LS-1348)

> Create issue template for nacelle-js

### WHAT is this pull request doing?

Adds two [GitHub Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms):

1. For [**Bug Reports**](https://github.com/getnacelle/nacelle-js/blob/LS-1348-issue-forms/.github/ISSUE_TEMPLATE/bug_report.yml)
2. For [**Feature Requests**](https://github.com/getnacelle/nacelle-js/blob/LS-1348-issue-forms/.github/ISSUE_TEMPLATE/feature_request.yml)

This PR also pares down our contribution docs, as the bug report and feature enhancement guidelines are taken care of in the Issue Forms.

## 

### How to Test

Check out the links in **1** and **2** above. I don't think there's a way to take these forms for a dry run before this is merged.

Note that the form metadata visible in the links above isn't visible to users when submitting a new issue with the form:

GitHub Issue Form metadata

### Testing checklist

* [x] Updated the `README.md` with documentation changes

### Next Steps

It might be wise to create a GitHub Action that takes care of adding new packages / examples / starters to the `project` section of the Issue forms. In the meantime, it's not a big deal to maintain this list by hand (it's not changing quickly yet).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,4 @@ npx @nacelle/create your-project-name
 
 ## How to Contribute
 
-- [Submit a Pull Request](./CONTRIBUTING.md)
-- [Report bugs](./BUG_REPORT.md)
-- [Suggest an enhancement](./SUGGEST_ENHANCEMENT.md)
+Please see our [Contribution Guidelines](./CONTRIBUTING.md) for more information about submitting bug reports, feature requests, and code contributions.
```
Comment (mdarrik) [inline README.md:33]: Wondering if we should move the Bug/Feature request stuff here directly? There's quite a bit in the contributing guide so people might not necessarily navigate all the way there? If not, we should definitely link directly to the bug reports & feature requests sections so people can be taken right to where they need.

**Review by mdarrik — APPROVED** [review]
Small typo and 1 possible suggestion. Otherwise LGTM

File: README.md (hunk 1 above)
Comment (NWRichmond) [reply README.md]: Great suggestions! Both have been implemented in https://github.com/getnacelle/nacelle-js/pull/47/commits/ded0574526462ea624b3cc66748ced363e1582d8.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -120,63 +120,37 @@ After creating `examples/<project-name>`, add the following script to its `packa
...
-### Reporting Bugs
+### Reporting bugs
 
-We use [Github Issues](https://guides.github.com/features/issues/) to track all reported bugs. Before submitting a bug we ask that you review our pre-bug [checklist](#before-submitting-a-bug).
+We use [Github Issues](https://guides.github.com/features/issues/) to track all reported bugs. Before submitting a bug we ask that you review the following pre-bug checklist:
 
 #### Before Submitting A Bug Report
 
-- **Reproduce the bug**. To eliminate any outside factors you should create an isolated code sandbox confirming the bug. The sandbox should include the minimal amount of code to reproduce the bug.
+- **Reproduce the bug**. To help rule out outside factors, you should create an isolated [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) that demonstrates the bug. We recommend tools like [Code Sandbox](https://codesandbox.io) and [StackBlitz](https://stackblitz.com) for minimal creating a minimal reproduction.
```
[the remark this answers is not in the document]
Comment (NWRichmond) [reply CONTRIBUTING.md]: Thanks for catching that! Fixed in https://github.com/getnacelle/nacelle-js/pull/47/commits/4fb04b683ec1255541132b68f9dd31248d257dd3.

**Pull request merged**